R/lancelot.R

```diff
@@ -390,6 +410,10 @@ lancelot_parameters <- function(start_date, region,
                                 vaccine_index_booster = NULL,
                                 vaccine_catchup_fraction = 1,
                                 n_doses = 2L,
```

Comment:
slightly unrelated to the PR but should we make n_doses = 3 the default now?

Reply:
Possibly, but not essential. Would likely require a lot of changes to tests so could be done in its own PR!